client/my-sites/hosting/web-server-settings-card/index.tsx

```diff
@@ -235,16 +251,6 @@ const WebServerSettingsCard = ( {
 		);
 	};
 
-	const changePhpVersion = ( event ) => {
-		const newVersion = event.target.value;
-
-		setSelectedPhpVersion( newVersion );
-	};
-
-	const updateVersion = () => {
-		updatePhpVersion( siteId, selectedPhpVersion );
-	};
-
```

Comment:
👍 I like the inlining of these. The state setter/selector involved are readable enough that it makes more sense that way